dev/script/generate-dot-properties/config.json5

```diff
@@ -21,34 +11,6 @@
     replicas: 0
   },
 
-  thrall: {
-    streamName: 'media-service-DEV-thrall',
-  },
-
-  // Guardian specific properties for the Usage service
```

Comment:
These config props have moved to sit under a `guardian` key for further clarity.